index.html

```diff
@@ -575,7 +575,7 @@
 
     <p>Contracts are a concept used in Eiffel to avoid bugs. Although these should be disabled in a production runtime, during development they can be quite useful. There are three types of assurance elements: preconditions (used in features), postconditions (used in features) and class invariants.</p>
 
-    <p>Preconditions are defined using the <b>require</b> keyword. They should contain a tag and a boolean expression. Postconditions are written the same way, but we use the <b>ensure</b> keyword. We can use the <b>old</b> notation to compare a variables value to it's value before the feature executed. Here is an example that might be used in the <b>BAKERY</b> class we saw above:</p>
+    <p>Preconditions are defined using the <b>require</b> keyword. They should contain a tag and a boolean expression. Postconditions are written the same way, but we use the <b>ensure</b> keyword. We can use the <b>old</b> notation to compare a variables value to its value before the feature is executed. Here is an example that might be used in the <b>BAKERY</b> class we saw above:</p>
```

Comment:
Thanks for the corrections! 😃 I'd just use 'was' instead of 'is', since postconditions are checked after the feature was executed.